src/index.html

```diff
@@ -22,6 +22,335 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__item">
+          <li>
```

Comment:
Add classes for tag li everywhere
```suggestion
          <li class="nav__item">
```